src/backend/joanie/tests/core/api/organizations/test_contracts_signature_link.py

```diff
@@ -32,7 +32,7 @@ def test_api_organization_contracts_signature_link_without_owner(self):
             payment_schedule=[
                 {
                     "amount": "200.00",
-                    "due_date": "2024-01-17T00:00:00+00:00",
+                    "due_date": "2024-01-17",
```

Comment:
Elsewhere, you are passing `Money` and `date` objects. You should do that also here.

Reply:
Elsewhere, we were testing what were the argument passed in to mocked function with the method `assert_called_once_with`.

The aim of this PR is when we create with the factories (OrderFactory, OrderGeneratorFactory) the payment schedule of an order, we can pass string and it will make sure that they casted in the right object in output.